start/sw/web-app/laravel5.md

```diff
@@ -71,7 +78,27 @@ DB_PASSWORD=Chur1
 
 #### Serve
 
+###### Do not run valet in sudo
 ```bash
 $ php artisan serve
 # Laravel development server started on http://localhost:PORT/
+
+# or use valet
+$ valet park
+$ valet link DOMAIN
+
+# Now visit http://DOMAIN.test/
+```
+
+###### if ssl is desired
+
+```bash
+$ valet secure
+# Now visit https://DOMAIN.test/
+
+```
+
+###### if need a callback URL/share work
```

Comment:
Looking at the <hX> elements of markdown, I can see you're only skipping the odd ones(1,3,5...). Why wouldn't you start from `# Header 1` and continue with `## Header 2`, `### Header 3`...?

 - https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet